upnext/create_party.py

```diff
@@ -58,15 +57,15 @@ def create_playlist(request, party_name):
                         track.save()
 
             # Add all the tracks to Spotify in order
-            for track in party.track_set.order_by('-score', 'track_title', 'artist'):
-                token_info = tokens.token_read()
+            token_info = tokens.token_read()
 
-                username = 'up--next'
-                track_id = track.uri.split(':')[-1]
-                party_id = party.uri.split(':')[-1]
+            username = 'up--next'
 
-                sp = spotipy.Spotify(auth=token_info['ACCESS_TOKEN'])
-                sp.user_playlist_add_tracks(username, party_id, [track_id])
+            tracks = party.track_set.order_by('-score', 'track_title', 'artist')
+            track_uris = map(lambda x: x.uri, tracks)
+
+            sp = spotipy.Spotify(auth=token_info['ACCESS_TOKEN'])
+            sp.user_playlist_add_tracks(username, party.uri, track_uris)
```

Comment:
Got rid of `party_id` and just used `party.uri`. Was there any particular reason we were doing the splitting before?

(`party.uri` is something like `spotify:user:up--next:playlist:501a0n8wr8u2GAzQpELW0L`,
while `party_id` is something like `501a0n8wr8u2GAzQpELW0L`)